assets/app/features/upgrade/i18n/en.json

```diff
@@ -32,50 +32,50 @@
+                        "description-title": "IMPORTANT!",
+                        "description1": "In normal mode, the workloads running in each node will be live-migrated to another node. Note that you will lose connection to your VMs when upgrading the control node(s).",
+                        "description2": "In order to proceed in non-disruptive mode your cloud needs to be Highly Available and have enough free compute resources. See the deployment guide for details.",
                         "description3": "Be sure to re-run the checks if you make any changes to your cloud after the initial run."
                     },
                     "nondisruptive": {
                         "title": "Your cloud is now prepared to begin the upgrade process in non-disruptive mode.",
-                        "description1": "In non-disruptive mode all workloads should continue running but we strongly recommend picking your nose and wiggeling your finger at it for a while first.",
+                        "description1": "In non-disruptive mode all workloads should continue running and remain connected.",
                         "description2": "Be sure to re-run the checks if you make any changes to your cloud after the initial run."
                     }
                 }
             },
             "backup": {
-                "name": "Back up Admin Server",
-                "title": "Create a backup of the Administration Server configuration and download it in your browser.",
-                "description": "Keep this backup somewhere safe in case of emergency. With this backup and a fresh deployment of SUSE OpenStack Cloud 6 your cloud can be returned to it’s previous state.",
+                "title": "Create a backup of the Administration Server configuration and download it in your browser",
+                "description": "This backup contains information about the cloud deployment and configuration. Keep this backup somewhere safe in case of emergency.",
                 "form": {
                     "backup": "Download Backup of Administration Server"
                 }
             },
             "admin-repository-checks": {
-                "name": "",
-                "title": "Upgrading to SUSE OpenStack Cloud 7 requires software repositories for SLES12 SP2 and SUSE OpenStack Cloud 7.",
-                "description": "Update the URLs of the following software repositories and click 'Check' to verify the repositories are available. When all repositories have been verified click on 'Next' to proceed.",
+                "title": "Upgrading to SUSE OpenStack Cloud 7 requires updated software repositories for the Administration Server",
+                "description": "Prepare the software repositories locally or via SMT and update the URLs in the Administration Server configuration correspondingly. Click \"Check\" to verify all repositories are available. When all repositories have been verified click on \"Next\" to proceed.",
```

Comment:
It might be good to add a sentence in a sense *Refer to **Software Repository Setup** chapter of the deployment guide.*